src/Core/Components/Dialog/Services/IDialogService-Dialog.cs

```diff
@@ -67,6 +69,14 @@ Task<IDialogReference> ShowPanelAsync<TDialog>(object data, DialogParameters par
     Task<IDialogReference> ShowPanelAsync<TDialog>(DialogParameters parameters)
         where TDialog : IDialogContentComponent;
 
+    /// <summary>
+    /// Shows a panel with the the RenderFragment as the body
+    /// </summary>
+    /// <param name="renderFragment">Body of the panel</param>
+    /// <param name="dialogParameters">Parameters to configure the dialog component.</param>
+    /// <returns></returns>
+    Task<IDialogReference> ShowDialogAsync(RenderFragment renderFragment, DialogParameters dialogParameters);
```

Comment:
Correct the comments to pertain to Dialog please

Reply:
Fixed